js/whisper-demo/index.js

```diff
@@ -40,6 +40,12 @@ function updateConfig() {
         if (pair[0] == 'maxChunkLength') {
             options.maxChunkLength = parseFloat(pair[1]);
         }
+        if (pair[0] == 'chunkLength') {
+            options.chunkLength = parseFloat(pair[1]);
+        }
+        if (pair[0] == 'maxAudioLength') {
+            options.maxAudioLength = parseFloat(pair[1]);

```

Comment:
It's better to limit the max audio length to 30s, the model is trained with 30s audio.

Reply:
Fixed in https://github.com/Honry/onnxruntime-inference-examples/pull/4/commits/97559398918adde21cc53412f26c8626e5c559a0